fnirs_tapping/sub-03/nirs/sub-03_task-tapping_channels.tsv

```diff
@@ -0,0 +1,57 @@
+﻿name	type	source	detector	wavelength_nominal	units	low_cutoff	high_cutoff	description	sampling_frequency	status	status_description
+S1_D1 760	NIRSCWAMPLITUDE	S1	D1	760.0	V	0.0	3.90625	Near Infrared Spectroscopy(continuous wave)	7.8125	good	n/a
```

Comment:
Is the missing space in `Near Infrared Spectroscopy(continuous wave)` (before the bracket) intentional?

Reply:
Wow great, catch. Thanks @sappelhoff